From ce4d251b4d5327051e764cb0693f699c8e584d94 Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Tue, 15 Jan 2019 12:02:49 -0500
Subject: [PATCH] ibus: Add automake decorations to sed in appropriate paths

  Multiple vapidir paths are required to build ibus correctly,
  so decorations are added to the automake files to allow for
  sed'ing the appropriate path during do_configure_prepend.

  Upstream-status: Inappropriate [configuration]

  Signed-off-by: Brendan Kerrigan <kerriganb@ainfosec.com>

---
 engine/Makefile.am  | 1 +
 tools/Makefile.am   | 1 +
 ui/gtk3/Makefile.am | 1 +
 3 files changed, 3 insertions(+)

diff --git a/engine/Makefile.am b/engine/Makefile.am
index 86f0e2b..7e922b5 100644
--- a/engine/Makefile.am
+++ b/engine/Makefile.am
@@ -53,6 +53,7 @@ AM_VALAFLAGS = \
 	--vapidir=$(top_builddir)/bindings/vala \
 	--pkg=ibus-1.0 \
 	--target-glib="$(VALA_TARGET_GLIB_VERSION)" \
+	@EXTRA_AM_VALAFLAGS@ \
 	$(NULL)
 
 libexec_PROGRAMS = \
diff --git a/tools/Makefile.am b/tools/Makefile.am
index 5c18d3d..fe727c1 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -61,6 +61,7 @@ AM_VALAFLAGS = \
 	--pkg=posix \
 	--pkg=config \
 	--target-glib="$(VALA_TARGET_GLIB_VERSION)" \
+	@EXTRA_AM_VALAFLAGS@ \
 	$(NULL)
 
 bin_PROGRAMS = ibus
diff --git a/ui/gtk3/Makefile.am b/ui/gtk3/Makefile.am
index 667d51c..cdcbf5a 100644
--- a/ui/gtk3/Makefile.am
+++ b/ui/gtk3/Makefile.am
@@ -79,6 +79,7 @@ AM_VALAFLAGS = \
 	--pkg=config \
 	--pkg=xi \
 	--target-glib="$(VALA_TARGET_GLIB_VERSION)" \
+	@EXTRA_AM_VALAFLAGS@ \
 	$(NULL)
 
 if ENABLE_APPINDICATOR
